### ISSUE TITLE
[Help] Val loss increasing while Train loss decreasing

### ISSUE THREAD

**Issue by tudor-mare** [YongWookHa/swin-transformer-ocr#7]
Hello,

I am trying to train the model on a generated dataset (~10k images). I am following the same configuration as in the repo but what I see is that the model learns by the train samples but doesn't generalize. The images are from the same domain so I can't imagine why this would be happening.

![image](https://user-images.githubusercontent.com/34603830/153597057-8c16072f-6ada-448a-b41e-653925a25404.png)

I am following same augmentations as in the repo. I am thinking maybe the learning rate is too big and the model learns the most common label, could that be the case?

How long should I leave the model to train? I was expecting to see healthy learning (val loss < train loss) from the beginning. I have left the model to train for longer and I see the same results.

Any help would be appreciated, thanks.

**Comment by YongWookHa** [comment]
Hi @tudor-mare.

Thank you for having an interest in this repository.

I guess the reason the model wasn't able to generalize is that your dataset wasn't enough.

In my case, I used 4M phrase (in Korean). I think you should increase the amount of dataset to at least more than 100k.

**Comment by tudor-mare** [comment]
Hi, thank you for the quick response!

Okay, I will try what you have suggested.

Thank you!

**Comment by ntqnhanguyen** [comment]
> Hi @tudor-mare.
> 
> Thank you for having an interest in this repository.
> 
> I guess the reason the model wasn't able to generalize is that your dataset wasn't enough.
> 
> In my case, I used 4M phrase (in Korean). I think you should increase the amount of dataset to at least more than 100k.

how many epochs did you train with 4M phrase?
